### DISCUSSION TITLE
Responses

### DISCUSSION THREAD

**Discussion by kinlane** [postman-open-technologies/linting-rules#18] [General]
This is a discussion around the responses ruleset, keeping an ongoing record of what has gone into the ruleset.

**Comment by jwbanning** [comment]
Looking at this link > https://rules.linting.org//rules/response-delete-204-no-body/

This does not look right:
 "field": "cont3ent",

<img width="962" alt="Screen Shot 2021-11-30 at 7 58 38 PM" src="https://user-images.githubusercontent.com/2584258/144157960-863d7efc-e782-42ec-bcc1-cac2e82cbd27.png">

**Comment by jwbanning** [comment]
From the page https://rules.linting.org/, looking at the individual rules and their links, I do not see an entry for 
response-put-204-status-code: {
description: "A put operation should have a 204 response.",
message: "A put operation should have a 204 response.",
...
}
Here is the URL of the rule: 
Ihttps://rules.linting.org//rules/response-put-204-status-code/

**Comment by jwbanning** [comment]
On this page > https://rules.linting.org/ 
The description and message HTTP status do not match the title:

<img width="1012" alt="Screen Shot 2021-12-01 at 7 00 24 AM" src="https://user-images.githubusercontent.com/2584258/144238895-eff3bc6f-9838-4b3a-bb1d-d104bf043e27.png">
